src/tree_data.h

```diff
@@ -997,6 +997,20 @@ struct lyd_node_opaq {
     const struct ly_ctx *ctx;       /**< libyang context */
 };
 
+/**
+ * @brief Context term data node record.
+ */
+struct lyd_term_nodes_ext_rec {
+    const struct lyd_node_term *node;           /** pointer to the node itself */
+    const struct lyd_node_term **leafref_nodes; /** list of the leafref data nodes [sized array](@ref sizedarrays)).
```

Comment:
Explain what `leafref data nodes` are. Specifically, this is the list of leafrefs pointing to `node`, I believe.

Reply:
Adjusted